modules/releng.gradle

```diff
@@ -343,6 +344,80 @@ prepareMajorIncrement {
 	}
 }
 
+tagAppReleases {
+	description = "Gathers the Git IDs from all app changelog files and tags them as 'marketplace-appName-appVersion'."
+
+	doLast {
+		FileTree appChangelogFileTree = fileTree(dir: projectDir, includes: [".releng/apps/**/app.changelog"])
+
+		appChangelogFileTree.each {
+			appChangelog ->
+
+			String appName = appChangelog.getParentFile().getName()
+
+			String appChangelogTextFiltered = appChangelog.filterLine {
+				it =~ /app\.git\.id/
+			}
+
+			List appGitIDLines = appChangelogTextFiltered.readLines()
+
+			for (appGitIDLine in appGitIDLines) {
+				String appGitID = appGitIDLine.replaceFirst(/app\.git\.id-.+=/, "")
+
```

Comment:
If there's no relation between these 2 lines of code, you can remove this empty line.